Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+<table>
+  <tr>
+    <td style="min-width: 25em;">__TOC__</td>
+    <td style="vertical-align: top; padding-left: 1em;">
+'''STATUS: READY'''
+'''RRA Version supported: 2.5.0'''
+
+Any team uses a risk based methodology when making any decision.
+The Rapid Risk Asessment or Rapid Risk Analysis (RRA) process helps formalize these decisions and ensures that they're
+reproducible, consistent and easy to communicate.
+
+This document assumes that the reader is familiar with risk definition, and the Mozilla risk levels from the
+[Risk_management] documentation (i.e. read it first).
+
+The Enterprise Information Security Team (Infosec) maintains this document as a reference guide for security engineers
+and anyone interested in following our risk management processes.
+
+Updates to this page should be submitted to the [https://github.com/mozilla/wikimo_opsec/ source repository on github].
+Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/master commit history].
+
+<span style="float: right; padding-top: 3em;">[[File:OpSec.png|300px]]</span>
+    </td>
+  </tr>
+</table>
+
+= Rapid Risk Assessment =
+
+{| class="wikitable"
+|-
+! <span style="color:red;">'''WARNING'''</span>
+|-
+|
+The Rapid Risk Analysis/Assessment (RRA) is not a security review or a full threat-model. A threat model, review, pen-testing, etc. may
+however follow an RRA.
+The RRA strongly focuses on getting the maximum amount of actionable information in the shortest amount of time available.
+'''The RRA process is made to analyze and assess services'''. Analyzing processes, or individual pieces of data is not
+recommended through the RRA process.
+|}
+
+== Preamble ==
+
+Data is the most important item in risk management. Software, websites, infrastructure, networks and people handle, process, exchange and store data.
+
+The RRA focuses efforts to create a summary of the risks associated with our data. In particular it aims to be:
+
+* '''Quick!''' About 30 minutes to 60 minutes maximum.
+* '''Very high-level'''. Details are for threat models.
+* '''Concise, readable'''. Short table with clear risk levels.
+* '''Re-runnable'''. Can be run during the architecture phase, and re-run if the project is re-architected or new high level design decisions are made.
+* '''Collects''' risk impact, an approximate impact likelihood and data dictionary.
+
+This helps to make the following type of decisions:
+
+* Can this project use systems which are easier to use but provide a lower security level?
```

Comment:
"Is the security provided by a given platform appropriate to host a specific type of data?"